community/readline/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+for patch in readline80-00* ; do patch -p0 < "$patch"; done
```

Comment:
https://k1ss.org/style.txt 1800

Reply:
In this case the files are staying on the upstream official source, I would like to request a style guide exception.

Reply:
I meant breaking it onto multiple lines. See 1800.

Reply:
oh, okay

Reply:
I thought it was the glob, the problem